<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNewLinkWizard</class>
 <widget class="QWizardPage" name="QNewLinkWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>325</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>通信连接</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>连接编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>连接名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设备服务：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>连接接口：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PLC连续地址间隔：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="newlinkwizard_server_combobox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="newlinkwizard_linkport_combobox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="newlinkwizard_plcaddress_combobox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="newlinkwizard_device_combobox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="maxVisibleItems">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="newlinkwizard_linknum_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="newlinkwizard_linkname_lineedit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>制造商：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>系列：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="newlinkwizard_linkport_combobox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="newlinkwizard_linkport_combobox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newlinkwizard_linkport_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newlinkwizard_linkport_combobox</tabstop>
  <tabstop>newlinkwizard_linkport_combobox_2</tabstop>
  <tabstop>newlinkwizard_linkport_combobox_3</tabstop>
  <tabstop>newlinkwizard_linkport_lineEdit</tabstop>
  <tabstop>newlinkwizard_linkname_lineedit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
